src/index.js

```diff
@@ -60,15 +61,25 @@ function observeDomChanges(contentClassName, productTileClassName, tileProp) {
   observer.observe(document.body, { subtree: true, childList: true });
 }
 
-function applyBoycott(productTileClassName, tileProp) {
+function applyBoycott(productTileClasses, productTileTags, tileProp) {
   const matchedBrands = new Set();
-  const productTiles = document.getElementsByClassName(productTileClassName);
+  let productTiles = []
+  for (const tileClass of productTileClasses) {
+    productTiles.push(...document.getElementsByClassName(tileClass));
+  }
+  if (productTileTags !== undefined) {
+    for (const tileTag of productTileTags) {
+      productTiles.push(...document.getElementsByTagName(tileTag));
```

Comment:
Is this necessary? Could you please remove the productTileTags addition in favour of utilising the productTileClasses for the same purpose? i.e. ".productContainer > li" instead of adding all "li" elements by tag name.